recipes/cython-cmake/recipe.yaml

```diff
@@ -0,0 +1,58 @@
+context:
+  name: cython-cmake
+  version: 0.2.0
+  username: scikit-build
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com//${{ username }}/${{ name|lower }}/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 0673545871646e1736a2aaaf29f0f96ad1033da8fa6e457ee1cb99289048b8ac
+
+build:
+  script: ${{ PYTHON }} -m pip install --no-build-isolation --no-deps -vv .
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.9
+    - hatchling
+    - hatch-vcs
+    - pip
+  run:
+    - python >=3.9
+    - cmake >=3.21
```

Comment:
Does this work? I think you can't put `cmake` in the `host` table, but if you depend on `cmake` and need to put `cython-cmake` in the `host` section (due to depending on Python, I believe this dep is correct), this would cause `cmake` to be included in host.

Reply:
OK. I will follow the specifications for `scikit-build-core`.